packages/core/src/commands/request-live-channel-command.class.ts

```diff
@@ -0,0 +1,22 @@
+import { Command, CommandResult } from '../framework/commands';
+import { ReactionResults } from '../framework/reactions';
+import { Reducer } from '../framework/reducers';
+
+export interface RequestChannelCommandRequestParameters {
+  reducer: Reducer;
+  startFromEID?: number;
```

Comment:
So... the past code had EID littered everywhere. I got stumped on this too. It means "eventId". You'll see that I did correct some of them but I left some alone since I figured I would get to it later (we're redoing channels anyway). I 100% agree with you - EID is so bizarre I think they all need to be removed.